src/main/java/io/split/android/client/service/sseclient/sseclient/SseHandler.java

```diff
@@ -60,7 +66,10 @@ public void handleIncomingMessage(Map<String, String> values) {
                 case SPLIT_KILL:
                 case SPLIT_UPDATE:
                 case MY_SEGMENTS_UPDATE:
-                    mNotificationProcessor.process(incomingNotification);
+                    if(!isStreamingPaused.get()) {
```

Comment:
nice catch! Please Javi, recheck if we included it in iOS, or consider it in some of the coming PRs.

Reply:
Using notification keeper now. Thanks doc!